### DISCUSSION TITLE
Do need to use mino client for storage backends?

### DISCUSSION THREAD

**Discussion by thotz** [milvus-io/milvus#31460] [Q&A and General discussion]
I can see configuration for minio client used by storage backends like `gcp` and `aliyun`. I am planning to add storage backend for [ceph-rgw](https://github.com/ceph/ceph/). Can we use [aws s3 client go SDK](https://docs.aws.amazon.com/sdk-for-go/api/service/s3/), and if possible use a common struct for both?

**Comment by LoveEachDay** [comment]
@thotz For now yes, pr is welcome.
